### DISCUSSION TITLE
"Template: One or more dynamic variables are corrupted "

### DISCUSSION THREAD

**Discussion by xnyjyh** [emailjs-com/help#156] [Q&A]
I get this in my email when I get a new signup with my form "Template: One or more dynamic variables are corrupted"

I'm supposed to get a detail emailed whenever someone submits my form. But I don't get it just that Template Corruption error message in my email. All my NAME are correct to match my form.

This is my email format below... Why am I not getting it properly?:

UNPOOPED has a new sign-up! Here are the details:

Personal Information
First Name: {{firstName}}
Last Name: {{lastName}}
Phone Number: {{phoneNumber}}
Email Address: {{email}}
Home Address: {{homeAddress}}
City: {{city}}
Province: {{province}}

Service Details
Number of Dogs: {{dogCount}}

Dog's Information
{{#if dog1Name}}

Dog 1 Name: {{dog1Name}}
Is it safe for us to be in the yard with your dog? {{dog1Safety}}
Additional comments about your dog: {{dog1Comments}}
{{/if}}
{{#if dog2Name}}

Dog 2 Name: {{dog2Name}}
Is it safe for us to be in the yard with your dog? {{dog2Safety}}
Additional comments about your dog: {{dog2Comments}}
{{/if}}
{{#if dog3Name}}

Dog 3 Name: {{dog3Name}}
Is it safe for us to be in the yard with your dog? {{dog3Safety}}
Additional comments about your dog: {{dog3Comments}}
{{/if}}
{{#if dog4Name}}

Dog 4 Name: {{dog4Name}}
Is it safe for us to be in the yard with your dog? {{dog4Safety}}
Additional comments about your dog: {{dog4Comments}}
{{/if}}
Service Preferences
Cleanup Frequency: {{frequency}}
Discount (if applicable): {{discount}}
Where is your gate located? {{gateLocation}}
Cleanup Notifications: {{cleanupNotification}}
Notification Type: {{notificationType}}
How did you hear about us? {{referralSource}}

**Comment by xnyjyh** [comment]
Hellooo?

**Comment by xr0master** [comment]
The syntax is incorrect. Please use official documentation
https://www.emailjs.com/docs/user-guide/dynamic-variables-templates/
